paasta_tools/cli/cmds/logs.py

```diff
@@ -100,9 +103,37 @@ def add_subparser(subparsers):
         default=DEFAULT_SOA_DIR,
         help="define a different soa config directory",
     )
+
+    status_parser.add_argument(
+        '-a', '--ago', dest='time_ago',
+        help='How long ago to get logs from, uses friendly durations like 32m or 1w. Defaults to 30 minutes',
+    )
+    status_parser.add_argument(
+        '-t', '--time', dest='time',
+        help='The timestamp that -a/--ago gets the logs relative to. In ISO-8601 format. Defaults to right now. '
+             'For example: --time 2016-06-02T13:00:00 --ago 1d '
+             'would give you logs from 1st June at 1pm up till 2nd June at 1pm',
+    )
+    status_parser.add_argument(
+        '-l', '--lines', dest='line_count',
+        help='The number of lines to retrieve from the specified offset. May optionally be prefixed with a "+" or "-" '
+             'to specify which direction from the offset, defaults to "-100"',
+        type=int,
+    )
+    status_parser.add_argument(
+        '-lo', '--line-offset', dest='line_offset',
+        help='The offset at which line to start grabbing logs from. For example 1 would be the first line. Paired with '
+             '--count +100 would give you the first 100 lines of logs. Defaults to the latest line\'s offset',
+        type=int
+    )
     default_component_string = ','.join(DEFAULT_COMPONENTS)
     component_descriptions = build_component_descriptions(LOG_COMPONENTS)
-    epilog = 'COMPONENTS\n' \
+    epilog = 'TIME/LINE PARAMETERS\n' \
+             'The args for time and line based offsetting are mutually exclusive, they cannot be used together. ' \
```

Comment:
Is this enforced anywhere? I'd expect a change somewhere that did something like ``if time and line: raise ValueError('Cannot have both time and line')``